zulipterminal/ui_tools/boxes.py

```diff
@@ -529,6 +547,37 @@ def keypress(self, size: Tuple[int, int], key: str) -> str:
             self.model.controller.view.write_box.msg_write_box.set_edit_pos(
                 len(quote))
             self.model.controller.view.middle_column.set_focus('footer')
+        elif is_command_key('EDIT_MESSAGE', key):
+            if self.message['sender_id'] != self.model.user_id:
+                self.model.controller.view.set_footer_text(
+                        " You can't edit messages sent by other users.", 3)
+                return key
+            # Time since the message was sent by user
+            time_since_msg_sent = time() - self.message['timestamp']
+            # Time limit set by realm admin
+            edit_time_limit = self.model.initial_data[
+                    'realm_message_content_edit_limit_seconds']
```

Comment:
I would drop the two time-related statements down and break the `elif` below; this keeps these variables closer to where they're used.

Also, if the variables are clear enough, or there are hopefully-clear (!) error messages as here, I would drop the comments - I would prefer a blank line to these comments, as long as the code is sufficiently clear :)